src/System.Net.Sockets/tests/FunctionalTests/DualModeSocketTest.cs

```diff
@@ -633,6 +638,10 @@ public void Dispose()
                 try
                 {
                     server.Dispose();
+                    if (accepted != null)
+                    {
+                        accepted.Dispose();
```

Comment:
See above: while this is the right pattern, it won't test for how a lot of code is actually written.

If we fix this test, please add tests that will verify that disposing the accept socket handles are handled well by the GC finalizer (not only when properly Disposed by the application code).